### PR TITLE
Redirects for the DATS project

### PR DESCRIPTION
**Pull request by agbeltran**

### CODE REVIEW

File: dats/.htaccess [hunk 1]
```diff
@@ -0,0 +1,5 @@
+RewriteEngine on
+RewriteRule ^$ https://datatagsuite.github.io/doc [R=302,L]
+RewriteRule ^(.*)$ https://datatagsuite.github.io/context/$1 [R=302,L]
+RewriteRule ^(.*)$ https://datatagsuite.github.io/schema/$1 [R=302,L]
```
Comment (davidlehn) [inline dats/.htaccess]: The first catch-all rule will always override the second one.

Comment (davidlehn) [reply dats/.htaccess]: Based on the README maybe you just mean to have one catch all rule like:
```
RewriteRule ^(.*)$ https://datatagsuite.github.io/$1 [R=302,L]
```

Comment (agbeltran) [reply dats/.htaccess]: yeap, I see what you mean, thanks... will push the change

Comment (davidlehn) [reply dats/.htaccess]: You might have wanted to keep that first "^$" rule?

**Pull request merged**